.circleci/config.yml

```diff
@@ -9,20 +9,22 @@ parameters:
     type: string
     default: hmpps_typescript_notifications
 
+  node-version:
+    type: string
+    default: 14.17-browsers
+
 executors:
-  builder:
-    docker:
-      - image: 'cimg/node:14.17-browsers'
-    working_directory: ~/app
   integration-tests:
     docker:
```

Comment:
think you can now change this to use the `node_redis` executor from hmpps too :grin:

Reply:
Ha I saw that PR coming and that's why I didn't bother to parameterise this section in the first place! One more push